webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -799,11 +840,12 @@ private static TableAndHighlighting createTable(PdfWriter docWriter, Document do
             // Gah, I have no idea where this number is coming from, see
             // https://github.com/cubing/tnoodle/issues/124 for why we had to bump
             // it from 8 to 9.
-            int HEIGHT_MARGINS = 9;
+            // Pedro says: I changed this margin to 17, so that text doesn't go all the way to the top of the cell and touch the border
+            int HEIGHT_MARGINS = 17;
```

Comment:
The original comment seems to be vestigial now, lets remove it. I think your comment doesn't add much either (you're just defining what a margin is, so lets remove it as well).